plugin/environment.go

```diff
@@ -162,6 +164,27 @@ func (env *Environment) Statuses() (model.PluginStatuses, error) {
 	return pluginStatuses, nil
 }
 
+// Manifest returns a manifest for a given pluginId.
+// Returns ErrPluginNotFound if plugin is not found.
+func (env *Environment) Manifest(pluginId string) (*model.Manifest, error) {
+	plugins, err := env.Available()
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get plugin statuses")
+	}
+
+	for _, plugin := range plugins {
+		if plugin.Manifest == nil {
```

Comment:
nit: a single check to `continue` here seems more verbose than necessary. Perhaps just:
```go
if plugin.Manifest != nil && plugin.Manifest.Id == pluginId {
    return plugin.Manifest, nil
}
```